index.html

```diff
@@ -395,6 +395,25 @@ <h2>Convert a <var>name</var> to a <var>timestamp</var></h2>
         The <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface was defined in [[NAVIGATION-TIMING]] and is now considered obsolete. The use of names from the <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface is supported to remain backwards compatible, but there are no plans to extend this functionality to names in the <a data-cite="NAVIGATION-TIMING-2#dom-performancenavigationtiming">PerformanceNavigationTiming</a> interface defined in [[NAVIGATION-TIMING-2]] (or other interfaces) in the future.
       </p>
     </section>
+  </section><section id="recommended-mark-names" class="informative">
+    <h2>Recommended Mark Names</h2>
+    <p>Developers are encouraged to use the following Recommended Mark Names to
+      mark common timings. The user agent is responsible for storing a new
+      timestamp under the specified mark name for Recommended Mark Names, just
+      like any user-specified mark name. The user agent does not validate that the
+      usage of the Recommended Mark Name is appropriate or consistent with its
+      description.</p>
```

Comment:
I think it's worthwhile to add a not indicating why we recommend people do this. Maybe something like:
```<p class=note>Adding such Recommended Mark Names can help RUM providers and browsers to collect web developer signals regarding their application's performance at scale, and surface this information to developers without requiring any site-specific work</p>``` (or something to that effect)

Reply:
Good point! Done.